packages/Organisms/HeroBanner/IntroBanner.vue

```diff
@@ -0,0 +1,158 @@
+<template>
+  <div class="rpl-intro-banner" :class="{ 'rpl-intro-banner--no-links': !showLinks }">
+    <div class="rpl-row">
+      <div class="rpl-intro-banner__left">
+        <h2 v-if="title" class="rpl-intro-banner__title"><span>{{ title }}</span></h2>
+        <p v-if="introText" class="rpl-intro-banner__description">{{ introText }}</p>
+      </div>
+      <div class="rpl-intro-banner__right" v-if="showLinks">
+        <h2 v-if="linkHeading" class="rpl-intro-banner__link-heading">{{ linkHeading }}</h2>
+        <ul class="rpl-intro-banner__link-list" v-if="links">
+          <li
+            v-for="(item, index) of links"
+            :key="index"
+            class="rpl-intro-banner__link-list-item"
+          >
+            <rpl-text-link
+              class="rpl-intro-banner__link"
+              :text="item.text"
+              :url="item.url"
+              iconSymbol="arrow_right_primary"
+              iconColor="primary"
+              :underline="true"
+              :theme="theme"
+            />
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { RplTextLink } from '@dpc-sdp/ripple-link'
+
+export default {
+  name: 'RplHeroBanner',
+  props: {
+    title: String,
+    introText: String,
+    linkHeading: String,
+    links: Array,
+    showLinks: { type: Boolean, default: true }
+  },
+  components: {
+    RplTextLink
+  }
+}
+</script>
+
+<style lang="scss">
+  @import "~@dpc-sdp/ripple-global/style";
+
+  $rpl-intro-banner-title-typography-ruleset: ('mega', 1.2em, 'bold') !default;
```

Comment:
Hi @alan-cole as most of intro banner is a reuse of hero banner style, I would like to avoid duplicated css. If we want to update the design in the future, we need to modify both css which will make maintain the consistent hard. Can you make them share the same variables and css?